FAQ.md

```diff
@@ -22,23 +22,34 @@
 #### What metrics are exposed by the metrics server?
 
 Metrics server collects resource usage metrics needed for autoscaling: CPU & Memory.
-Metric values use standard kubernetes units (`m`, `Ki`), same as those used to
-define pod requests and limits (Read more [Meaning of CPU], [Meaning of memory])
+Metrics values use [Metric System prefixes] (`n` = 10<sup>-9</sup> and `Ki` = 2<sup>10</sup>),
+similar to those frequently used to define pod requests and limits (`m` = 10<sup>-3</sup> and `Mi` = 2<sup>20</sup>).
 Metrics server itself is not responsible for calculating metric values, this is done by Kubelet.
 
+[Metric System prefixes]: https://en.wikipedia.org/wiki/Metric_prefix
+
 #### How CPU usage is calculated?
 
-CPU is reported as the average usage, in CPU cores, over a period of time.
+CPU is reported as the average core usage measured in cpu units. 
+One cpu, in Kubernetes, is equivalent to 1 vCPU/Core for cloud providers and 1 hyperthread on bare-metal Intel processors.
+
 This value is derived by taking a rate over a cumulative CPU counter provided by the kernel (in both Linux and Windows kernels).
-The kubelet chooses the window for the rate calculation.
+Time window is used to calculate CPU is exposed under `window` field in Metrics API.
+Read more about [Meaning of CPU].
```

Comment:
We should add a new line before this one to avoid having it appended to the previous paragraph about the time window since the article does not mention anything about it and is more general to CPU calculation.

Reply:
done